pigeons/instabug.api.dart

```diff
@@ -1,5 +1,14 @@
 import 'package:pigeon/pigeon.dart';
 
+@FlutterApi()
+abstract class FeatureFlagsFlutterApi {
+  void onW3CFeatureFlagChange(
```

Comment:
Let's use the actual name and change W3* members to W3C*  for all of the file content.